proto/attestation/reference_value.proto

```diff
@@ -46,57 +47,35 @@ message VerifyAmdSevAttestationReport {
   // If true, will skip the check that the TEE is not in debug mode.
   bool allow_debug = 3;
 
-  optional VerifyLogEntry stage0_binary = 4;
+  optional LogEntryVerification stage0_log_entry = 4;
 }
 
-message VerifyIntelTdxAttestationReport {
+message IntelTdxReferenceValues {
   // TBD
 }
 
 // Verifies the log entry, including signatures and the digest.
-message VerifyLogEntry {
+message LogEntryVerification {
   // The endorser’s public signing key for signature verification.
   bytes endorser_public_key = 1;
 
   // Rekor’s public signing key for signature verification.
   bytes rekor_public_key = 2;
 }
 
-// Verifies that the field contains at least one of the given digests.
-// No checks are performed if this is empty.
-message Measurements {
-  repeated RawDigest digests = 1;
+// As measured by stage0
+message KernelLayerReferenceValues {
+  optional LogEntryVerification log_entry = 1;
+  optional RawDigest kernel = 2;
+  optional string kernel_cmd_line = 3;
+  optional RawDigest kernel_setup_data = 4;
+  optional RawDigest init_ram_fs = 5;
+  optional RawDigest memory_map = 6;
+  optional RawDigest acpi = 7;
```

Comment:
Shouldn't these be repeated? Or use some other intermediate message to represent `any of these digests`?

Reply:
Done.